java/code/src/com/redhat/rhn/domain/action/virtualization/VirtualizationCreateAction.java

```diff
@@ -0,0 +1,261 @@
+/**
+ * Copyright (c) 2018 SUSE LLC
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package com.redhat.rhn.domain.action.virtualization;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ *
+ * VirtualizationCreateAction - Class representing TYPE_VIRTUALIZATION_CREATE
+ */
+public class VirtualizationCreateAction extends BaseVirtualizationAction {
```

Comment:
Do we need to override the `equals` method since we added fields?

Reply:
I didn't need it so far, also overrode `hashCode()`.